backend/prisma/schema.prisma

```diff
@@ -7,31 +7,110 @@ generator prisma_client {
   provider = "prisma-client-js"
 }
 
-enum Status {
-  ADMIN MODERATOR FREE_USER PRO_USER
+model User {
+  Id                String        @id @default(cuid())
+  Name              String
+  Email             String        @unique
+  Password          String
+  ResetToken        String?
+  ResetTokenExpiry  Float?
+  userRole          UserRole      @default(FREE_USER)
+  CreatedAt         DateTime      @default(now())
+  UpdatedAt         DateTime      @updatedAt
+  Handle            String        @unique
+  Bio               String
+  SocialMedia       SocialMedia[]
+  interests         Topic[]
+  location          Location?
+  languagesNative   Language[]    @relation("LanguageToNative")
+  languagesLearning Language[]    @relation("LanguageToLearner")
 }
 
-model User {
-  Id               String   @id @default(cuid())
-  Name             String
-  Email            String   @unique
-  Password         String
-  ResetToken       String?
-  ResetTokenExpiry Float?
-  status           Status[]
-  posts            Post[]
-  CreatedAt        DateTime @default(now())
-  UpdatedAt        DateTime @updatedAt
+model SocialMedia {
+  Id       String              @id @default(cuid())
+  Platform SocialMediaPlatform
```

Comment:
Yeah I haven't finished building some of these out just yet so it might be a little confusing, just wanted to post so that you can see what I'm working on and the progress 😄 The `SocialMedia` model is for storing information about a particular platform. Users will have the option to add their social media info if they would like to, so they might add their LinkedIn and Instagram for example; or perhaps they have a YouTube channel. This will end up being a clickable icon that leads to the corresponding page/link. 

We're specifically supporting Facebook, YouTube, Instagram, and LinkedIn. We may have it in the code so that when they go to write in their link, it pre-populates most of the link so they only have to add their unique portion. Take a look at the model now with the Url field 😊🔗

Reply:
Ok that makes sense! So they could just add `nicklitwin` and then we'd construct the url to be `facebook.com/nicklitwin` 👍 

Is the Url field added to `SocialMedia`? I couldn't find it, but maybe it's just not pushed up yet?

Reply:
Yep it's there, did you look at the files changed? It doesn't show updates to the comment preview. It's currently on line 32 🙂

Reply:
Doh! Not sure how I missed that. looks good to me :)